extensions/ql-vscode/test/pure-tests/remote-queries/markdown-generation/data/interpreted-results/problem/expected/meteor-meteor.md

```diff
@@ -46,3 +46,26 @@ pp.strictDirective = function(start) {
 *This part of the regular expression may cause exponential backtracking on strings starting with '"' and containing many repetitions of '\!'.*
 
 ----------------------------------------
+
+[app/src/main/AndroidManifest.xml](https://github.com/AlexRogalskiy/android-nrf-toolbox/blob/034cf3aa7d2a3a4145177de32546ca518a462a66/app/src/main/AndroidManifest.xml#L239-L249)
```

Comment:
Note: I'm aware that this example doesn't make any sense in the context of the given repo and language (`meteor/meteor` and `javascript`) 😅 

But since it's just test data, I thought it would be easier to tweak an existing test than add a completely new example!